src/components/safe-apps/AppCard.tsx

```diff
@@ -73,6 +126,10 @@ const AppCard = ({ safeApp }: AppCardProps): ReactElement => {
     navigator.clipboard.writeText(shareUrl)
   }
 
+  if (variant === 'compact') {
```

Comment:
Please make `compact` and `default` an enum.

Reply:
Why? Then we must import it into every component that uses it if it is an enum. E.g., material UI doesn't use enums for component variants

Reply:
Importing the enum is the way, it's more robust.

Reply:
If it was a library, I would agree that plain strings are better. But typing out strings in our own code just seems weird.
It's easier and less error prone to autocomplete a symbol than a string literal. Anyway, not in the mood to argue. Keep the strings if you want to.